src/bilder/images/airbyte/deploy.py

```diff
@@ -75,47 +75,47 @@
 
 env_template_file = Path("/etc/consul-template/.env.tmpl")
 nginx_pomerium_conf_template_file = Path(
```

Comment:
This is already a `Path` object so it's not necessary to wrap it in the `Path` constructor